solutions/devsprint-roque-buarque-1/local.properties

```diff
@@ -1,10 +1,9 @@
-## This file is automatically generated by Android Studio.
-# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
-#
-# This file should *NOT* be checked into Version Control Systems,
+## This file must *NOT* be checked into Version Control Systems,
 # as it contains information specific to your local configuration.
 #
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-sdk.dir=/Users/rdgborges/Library/Android/sdk
\ No newline at end of file
+#Tue May 31 20:08:52 BRT 2022
+sdk.dir=C\:\\Users\\Rodrigo\\AppData\\Local\\Android\\Sdk
```

Comment:
não podemos subir o arquivo `local.properties` para o repositório, isso irá causar conflitos.
Veja esse [link](https://devconnected.com/how-to-remove-files-from-git-commit/) para remover o arquivo do seu commit.
Eu vou atualizar o gitignore para ignorar alguns arquivos e te aviso para dar um pull e atualizar sua branch.